ui/partials/perf/comparectrl.html

```diff
@@ -1,5 +1,5 @@
 <div class="container-fluid">
```

Comment:
Could you change the revision information header to handle this sort of case? Right now we're seeing this:

![image](https://cloud.githubusercontent.com/assets/20569/26558692/b6215c94-4478-11e7-89ee-3561d4e2bfe5.png)

I'd rather see something like:

![image](https://cloud.githubusercontent.com/assets/20569/26558733/fccff056-4478-11e7-9a45-8af2ca500891.png)